.mvx/demo.sh

```
@@ -151,12 +156,32 @@ if [ -n "$MAIN_CLASS" ]; then
-    eval java -cp \"$cp\" $JVM_OPTS -Dgosh.home=\"demo\" -Djava.util.logging.config.file=\"${logconf}\" ${MAIN_CLASS} $APP_ARGS
+
+  if [ "$demo_name" = "console" ]; then
+    # Console provider demo: use module path so System.console() picks up the JLine provider
+    mp=""
+    if [ -d ${TARGETDIR}/lib ]; then
+      mp=$(find ${TARGETDIR}/lib -name "*.jar" -exec printf :{} ';')
+      mp="${mp#:}"
+    fi
+    echo "Using module path for console provider"
+    eval java \
+      --module-path \"$mp\" \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)